test/new_relic/agent/transaction_sampler_test.rb

```diff
@@ -262,7 +262,8 @@ def test_harvest_slowest
         end
         @sampler.merge!([slowest])
         not_as_slow = @sampler.harvest![0]
-        assert((not_as_slow == slowest), "Should re-harvest the same transaction since it should be slower than the new transaction - expected #{slowest.inspect} but got #{not_as_slow.inspect}")
+        assert_equal(not_as_slow, slowest)
+        assert("Should re-harvest the same transaction since it should be slower than the new transaction - expected #{slowest.inspect} but got #{not_as_slow.inspect}")
```

Comment:
I would like someone to confirm that breaking up these asserts was the right move. It looked like we were doing multiple asserts in one `assert` statement. The tests passed locally when ran.

Reply:
The second assert isn't providing any value as-is. Originally the interpolated text was producing a message for a dev to review if the assertion failed. If you look up assert_equal's docs and it says it supports custom error messages, then you can adapt this to fit. Otherwise, just remove the second assert.

Reply:
Ah ok thank you!

Reply:
I think this is the message. Three arguments can be passed to the assertion in many cases, the expected value, the value being tested, and the error message to display if the test fails.

Reply:
Whoops! Missed that a review already came in! :)

Reply:
`assert_equal` supports customer error messages, so I [added it back](https://github.com/newrelic/newrelic-ruby-agent/pull/1482/files#diff-a14cc42262a47db44e26a23ad7af19b88326c9abbf25ddeab15c49d9a9f2a65bR265) :) thank you both!